charts/seed-monitoring/charts/grafana/dashboards/owners/kubernetes-control-plane-status-dashboard.json

```diff
@@ -2080,7 +2080,7 @@
       "type": "row"
     }
   ],
-  "refresh": "1m",
+  "refresh": "",
```

Comment:
Nit: I guess we could remove the field completely to prevent a confusion whether it gets defaulted to something when it is empty or not. At least removal is what we did with https://github.com/gardener/gardener/pull/4229.

```suggestion
```

Reply:
We explicitly selected in the Grafana UI to turn off the refresh, and this is how it renders in the json model, so I think it wouldn't default to a different setting. To be consistent with the other changes, I agree to remove it. 👍